script/glam/run_glam_sql

```diff
@@ -167,7 +167,7 @@ function run_desktop_sql {
         run_init "clients_histogram_aggregates_v1"
         run_query "clients_histogram_aggregates_new_v1"
         run_partitioned_query "clients_histogram_aggregates_v1" \
-            true "submission_date" "sample_id,app_version,channel"
+            true 1 "submission_date" "sample_id,app_version,channel"
```

Comment:
True, this was missing the the number of partitions but I think that parameter comes before the boolean param like on line `173` right?